add ram and cpu allocation ration

lachie83 opened the pull request.

lachie83 commented:
Only added to kilo at this point. Can you confirm that only the controller nodes need this change?

pupapaik commented:
It should be enough for nova scheduler only. See this http://www-01.ibm.com/support/knowledgecenter/SS8MU9_2.2.0/Admin/tasks/settingresourceovercommitratios.html

We need to configure scheduler_filters too.

lachie83 commented:
@pupapaik I've moved everything to metadata and included defaults, removed conditionals and updated example pillar in README. LMK if this looks better.

pupapaik commented on nova/files/kilo/nova-controller.conf.Debian:
```diff
@@ -20,6 +20,17 @@ libvirt_use_virtio_for_bridges=True
+ram_allocation_ratio = {{ controller.ram_allocation_ratio }}
+scheduler_default_filters = {{ controller.scheduler_default_filters }}
+
+{%- if controller.cpu_allocation_ratio is defined %}
```
There is still condition from the first commit. Please remove them.

lachie83 commented:
Sorry about that. Updated.

pupapaik commented:
Looks good to me. Thanks
